tests/tests_pytorch/strategies/test_deepspeed_strategy.py

```diff
@@ -1155,48 +1156,6 @@ def test_deepspeed_gradient_clip_by_value(tmpdir):
         trainer.fit(model)
 
 
-@RunIf(min_cuda_gpus=2, standalone=True, deepspeed=True)
-def test_specific_gpu_device_id(tmpdir):
```

Comment:
The behavior of these deleted test was that they would allocate the memory on the wrong GPU given by the local rank instead of the given device index (see output of nvidia-smi).